content/pages/community.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Community</title>
+        <meta name="save_as" content="community/index.html">
+        <meta name="url" content="community/">
+        <meta name="body_class" content="community_page">
+    </head>
+
+    <body>
+        <div class="community secondary">
```

Comment:
Convert this tag to `section` to make it consistent with other pages. Also the class order should be `col-12 secondary community`.